.github/workflows/docker/compose/guardrails-compose.yaml

```diff
@@ -9,3 +9,28 @@ services:
     build:
       dockerfile: comps/guardrails/llama_guard/docker/Dockerfile
     image: ${REGISTRY:-opea}/guardrails-tgi:${TAG:-latest}
```

Comment:
`guardrails-compose.yaml` include the images used by `GenAIExamples`, `guardrails-compose-cd.yaml` include the other images not used by `GenAIExamples`. No need to duplicate images in those 2 files.